src/index.html

```diff
@@ -1,14 +1,32 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
```

Comment:
why did you remove meta tag with viewport info?

Reply:
I was wrong, 
I thought formatting it's mean delete) everything is fixed